packages/common/src/events/_internal/EventBusinessRules.ts

```diff
@@ -33,8 +33,6 @@ export const EventPermissionPolicies: IPermissionPolicy[] = [
     permission: "hub:event",
     services: ["events"],
     licenses: ["hub-premium"],
-    // gating
-    environments: ["devext", "qaext"],
```

Comment:
Additionally, I have a question out to Brian to confirm the expectation in the following scenario:

As `juliana_px`:

1. opt into workspace
2. create an Events 3 event
3. publish a `Site|Page` with an `Events` card
4. opt out of workspace
5. view the `Site|Page` live view

I'm not sure if the user should see the `Events` card on the view since they are opted out of workspace. Will reply here once I have a clear understanding of Brian's expectations.